src/Hl7.Fhir.Conformance/Specification/Snapshot/ElementDefnMerger.cs

```diff
@@ -226,7 +226,7 @@ void onConstraint<T>(List<T> snap) where T : Element
                 {
                     if (snap.IsNullOrEmpty())
                     {
-                        result = (List<ElementDefinition.TypeRefComponent>)diff.DeepCopy();
+                        result = (List<ElementDefinition.TypeRefComponent>)diff.DeepCopyInternal();
```

Comment:
Why wouldn't this just call the extension method, like everyone else? The fact that it can call the internal function is incidental, this code might move to another assembly in the future.